### ISSUE TITLE
Ties between notes of chords sometimes cause verovio to crash

### ISSUE THREAD

**Issue by bbloomf** [rism-digital/verovio#2310]
**Describe the problem**
Running verovio on an MEI with tied notes within a chord sometimes causes a crash

**To Reproduce**
1. Load the data attached
2. Render page 1
3. See error (verovio crashes.  3.5.0 Javascript toolkit gives the following exception:
```
RuntimeError: function signature mismatch
    at wasm-function[2154]:0x14adfa
    at wasm-function[2155]:0x14b1bc
    at wasm-function[1917]:0x135e4a
    at wasm-function[369]:0x2a07e
    at wasm-function[348]:0x24235
    at wasm-function[654]:0x4be07
    at wasm-function[840]:0x5ce06
    at wasm-function[1260]:0x8c832
    at wasm-function[1849]:0xf6201
    at wasm-function[1183]:0x845db
```

**Expected behavior**
Verovio 3.3.0 produced valid SVG: [test.svg.zip](https://github.com/rism-digital/verovio/files/6878801/test.svg.zip)

**Input data**
[test.mei.zip](https://github.com/rism-digital/verovio/files/6878781/test.mei.zip)

**Verovio information**
 - Version of Verovio: 3.4.0, 3.4.1, 3.5.0
 - Binding: Javascript

**Environment information (as appropriate)**

 - OS: MacOS
 - Browser Chrome, Safari, NodeJS

**Additional context**
Pasting this MEI into https://editor.verovio.org/ causes the site to hang, and requires removal of browser cache / local storage to be able to load that site successfully again.

**Comment by lpugin** [comment]
It looks like it is crashing here
https://github.com/rism-digital/verovio/blob/6deab9316280d9eb7c715b2aab2742083121821e/src/tie.cpp#L237-L238

The search for `Dots` should be asserted, but that will obviously not fix the issue.

**Comment by lpugin** [comment]
It crashes because of the `dots="0". I fixed it and see if we should have a 3.5.1 release for that.
